utils/common/src/main/kotlin/PluginManager.kt

```diff
@@ -27,14 +27,14 @@ import java.util.ServiceLoader
 inline fun <reified T : Any> getLoaderFor(): ServiceLoader<T> = ServiceLoader.load(T::class.java)
 
 /**
- * An interface to be implemented by plugins that have a name.
+ * An interface to be implemented by any ORT plugin.
  */
-interface NamedPlugin {
+interface OrtPlugin {
```

Comment:
I thought we settled on just `Plugin` now, or? (In the context of renaming `OrtIssue` to `Issue`.)